Sources/Load.php

```diff
@@ -100,10 +100,6 @@ function reloadSettings()
 	if (empty($modSettings['force_ssl']))
 		$image_proxy_enabled = false;
 
-	// UTF-8 ?
-	$utf8 = (empty($modSettings['global_character_set']) ? $txt['lang_character_set'] : $modSettings['global_character_set']) === 'UTF-8';
-	$context['utf8'] = $utf8;
```

Comment:
The context variable may be referenced by mods.  Leave it alone with a hard coded value.

Reply:
Good point.  Addressed by latest commit.